CHANGELOG.md

```diff
@@ -3,6 +3,66 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.8.0
+
+_Jun 15, 2023_
+
+We'd like to offer a big thanks to the 13 contributors who made this release possible. Here are some highlights ✨:
+
+- 💫 Add a `referenceDate` prop on `DateCalendar`, `MonthCalendar` and `YearCalendar`.
+- 🌍 Add Greek (el-GR) locale
+- 🚅 Performance improvements
+- 🐞 Bugfixes
+- 📚 Documentation improvements
+
+### `@mui/x-data-grid@6.8.0` / `@mui/x-data-grid-pro@6.8.0` / `@mui/x-data-grid-premium@6.8.0`
+
+#### Changes
+
+- [DataGrid] Add missing styles to `overridesResolver` (#9248) @mrmuhammadali
+- [DataGrid] Avoid column headers hover on touch devices (#9076) @cherniavskii
+- [DataGrid] Correct the type for single digit edited number value (#9282) @MBilalShafi
+- [DataGrid] Filtering performance: cache values (#9284) @romgrk
+- [DataGrid] Fix tabbing between `actions` cells in edit mode (#9321) @md250721
+- [DataGrid] Make autocomplete work for `GridColDef['type']` (#9320) @cherniavskii
```

Comment:
```suggestion
- [DataGrid] Make `Autocomplete` work for `GridColDef['type']` (#9320) @cherniavskii
```

Reply:
Indeed, sorry for not looking into the PR itself. 🤦 
But on the other hand, maybe the PR title could be improved a bit..? 🤔 If it confused me, it might also confuse someone else. 🤷 
What do you think about this? @cherniavskii 
```suggestion
- [DataGrid] Fix type inference for `GridColDef['type']` (#9320) @cherniavskii
```

Reply:
Probably this will satisfy both your concerns 🙂 
```suggestion
- [DataGrid] Make autocompletion work for `GridColDef['type']` (#9320) @cherniavskii
```